src/app/inventory/store/d2-item-factory.service.ts

```diff
@@ -375,6 +381,8 @@ export function makeItem(
     source: itemDef.collectibleHash
       ? defs.Collectible.get(itemDef.collectibleHash).sourceHash
       : null,
+    collectibleState: collectible && collectible.state,
+    hasCollectible: !!collectible,
```

Comment:
Do we need this or can we just check if collectibleState is defined?

Reply:
yeah, im not even sure why i put that in. I'm not even using it